flink-ml-examples/src/main/java/org/apache/flink/ml/examples/feature/VectorAssemblerExample.java

```diff
@@ -56,7 +56,8 @@ public static void main(String[] args) {
         VectorAssembler vectorAssembler =
                 new VectorAssembler()
                         .setInputCols("vec", "num", "sparseVec")
-                        .setOutputCol("assembledVec");
+                        .setOutputCol("assembledVec")
+                        .setSizes(2, 1, 5);
```

Comment:
Let's update VectorAssembler's markdown document accordingly, including its parameter list, examples and possibly descriptions.